Wdi scripts

ccjoel opened the pull request:
Porting over world bank indicator populating scripts from "the other repo".

Uses poetry as most of the other recent utility `api/scripts`:

```
$ cd ./api/scripts/worldbank_upload_datasets
$ poetry shell
$ poetry install
$ poetry run main
```

And that's it! Wait for your indicators index to get filled with a lot of indicators.

hartleybrody commented:
🤠 decided to cowboy merge since it's a single commit, but let me know if it needs follow up changes!

ccjoel committed:
Initial import of wdi scripts from dojo-modeling org.

ccjoel committed:
Added output folder, will gitignore next. added downloads to gitignore. Partial
changes to script to use proper relative dirs.

ccjoel committed:
World bank script working. Onwards!

ccjoel committed:
changed global project gitignore so that we can add the country.csv asset file for wdi dojo es indicator populate script

ccjoel committed:
adding download folder with gitkeep

ccjoel committed:
renamed downloads folder to download so that it isnt ignored by top-level project gitignore glob

ccjoel commented:
@jgawrilo @adamgilg @Sorrento110 @brandomr:

If you ever need to populate your local dojo instance, you can use this world-bank data populate script by Kyle.
I ported this from the old dojo repo, changed to use poetry, and added to `api/scripts` folder.
Let me know if you wish to use it and have any questions! Hope it helps.
